.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: true
```

Comment:
This changes to the non-container based build environment which results in longer build times.

1) Is it necessary to have sudo access to support the proto compiler?
2) Could we move the building of the gateway into the Docker image itself? That way, the Docker container, which would have full root privileges could do all of the building and we wouldn't have to do anything else in our build other than call docker build from within the gateway directory?

Reply:
@prodonjs I made the changes, now its building the gateway on docker, i think this will simply travis build, i made the change about camelCase response message on pubsub.proto on gateway too, if you have suggestion let me know thanks.